src/app/games/game/game-faq/game-faq.component.ts

```diff
@@ -0,0 +1,24 @@
+import { KeyValue } from '@angular/common';
+import { ThrowStmt } from '@angular/compiler';
+import { Component, Input, OnChanges, OnInit } from '@angular/core';
+import { forkJoin } from 'rxjs';
+import { GamesService } from 'src/app/games/games.service';
+import { IGame } from 'src/app/games/games.service.interface';
+
+
+ @Component({
+  selector: 'game-faq',
+  templateUrl: './game-faq.component.html',
+  styleUrls: ['./game-faq.component.scss']
+})
+export class GameFAQComponent {
+  
+  constructor(private gamesService: GamesService) { }
+
+  
+  @Input() game: IGame = null;
```

Comment:
Same here

Reply:
I don't know if this does anything.

Reply:
If it doesn't, remove it :)

Reply:
It does something... stops it failing to compile.

Reply:
```
Error: src/app/games/game/game-faq/game-faq.component.html:2:43 - error TS2339: Property 'game' does not exist on type 'GameFAQComponent'.

2     <markdown [src]="'/assets/md/' + this.game.faq + '.md'">
                                            ~~~~

  src/app/games/game/game-faq/game-faq.component.ts:14:16
    14   templateUrl: './game-faq.component.html',
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component GameFAQComponent.


Error: src/app/games/game/game.component.html:7:13 - error NG8002: Can't bind to 'game' since it isn't a known property of 'game-faq'.
1. If 'game-faq' is an Angular component and it has 'game' input, then verify that it is part of this module.
2. If 'game-faq' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.
3. To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component.

7   <game-faq [game]="game"></game-faq>
              ~~~~~~~~~~~~~

  src/app/games/game/game.component.ts:11:16
    11   templateUrl: './game.component.html',
                      ~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component GameComponent.```
So it's needed to add `game` as an element of whatever `this` is?

Reply:
Needed to identify what markdown to display